### ISSUE TITLE
I will close this one, please feel free to reopen it.

### ISSUE THREAD

**Issue by thommierother** [go-gitea/gitea#17182]
I will close this one, please feel free to reopen it.

_Originally posted by @lunny in https://github.com/go-gitea/gitea/issues/5360#issuecomment-461802774_

**Comment by thommierother** [comment]
See closed issue 5360 for reference. I see the same issue now on the current Gitea-Version: 1.14.5:

```
inux-ssh: Linux ssh wrapper invoked with arguments: [git@gitea.netzwissen.de
linux-ssh: using SSH command [ssh]
fatal: 'xxxx/yyy.git' does not appear to be a git repository
fatal: Could not read from remote repository.
Please make sure you have the correct access rights
and the repository exists.
```

This appears either when using atom or using _git push_ on shell

**Comment by zeripath** [comment]
This is an absolutely inadequate issue report.

This is not how you reopen an issue and any issue in 1.14.5 is highly unlikely to have anything to do with a problem in Gitea version 1.1. (Even if it was related the previous report is so completely out-of-date that it would be better to simply reference it saying this appears to be similar to #5360.)

Please open a new issue with a complete proforma - telling us version information, setup information and giving us logs.

**Comment by thommierother** [comment]
nevertheless, your harsh reaction is also inadequate ;-((

**Comment by zeripath** [comment]
You opened a bug report that is literally titled: "I will close this one, please feel free to reopen it."

The body of which is:

```
I will close this one, please feel free to reopen it.

_Originally posted by @lunny in https://github.com/go-gitea/gitea/issues/5360#issuecomment-461802774_
```

Requiring unpaid volunteers to follow a link and then trawl through a bug-report closed over **2** years ago to discover that it was referring to an issue in Gitea 1.1 - we are now on Gitea 1.15.3. Let's look at the differences between those versions: 

```
$ git diff --shortstat v1.1.0 v1.15.3
 12947 files changed, 2447076 insertions(+), 543551 deletions(-)
```

That is over 2 million lines of code different.

You've then posted a second comment that is not much more helpful and barely gives any more information. For example how are you running Gitea? Do you see anything in the logs?

---

We're genuinely happy to help but please help us to help you.

Please just open a new bug report with a proper title giving complete information about what happened and giving us logs.

https://github.com/go-gitea/gitea/issues/new?assignees=&labels=&template=bug-report.yaml

In your report reference #5360 if you believe that it's similar to your problem. You may well be right, but that bug is so old and so out of date that we need to start from scratch with full-information if we're ever going to solve it.

**Comment by thommierother** [comment]
sorry, you are correct, the incorrect issue title was my fault, I changed it but did not save the change correctly. Anyway, the issue is solved after an upgrade from 1.14 to the gitea 1.15.2 build.
